src/main/java/seedu/address/model/Model.java

```diff
@@ -125,6 +126,9 @@
 
     void addRooms(int num);
 
+    boolean canFit();
```

Comment:
will it be possible to add in java docs for these methods?

Reply:
Okay.